### ISSUE TITLE
(Discussion) Do we use tags?

### ISSUE THREAD

**Issue by jeroentskui** [cherrytwist/Server#4696]
we have tags in ltos of places; but we dont have search in lots of places

What happens when you press on a tag, (what should happen, how do we want them used, etc)

**Comment by keyboardguy25** [comment]
suggestions: if I click on a tag, I would like to see all related public posts / accounts / spaces that use that tag (or private posts / accounts / spaces that I have the rights to).

**Comment by keyboardguy25** [comment]
tag search: I included the tag "marketing" on my profile (under "Skills"), but when I search for "marketing" as a keyword in general search, my profile does not appear. This would make sense if my profile were private, but we don't have that option, so if I add a tag to my profile, I should show up in search with that tag.

if I look for "communications", which I added under "Keywords", I find my profile. Shouldn't it be better streamlined to have 1 tag section for everything? or, if the sections stay split, couldn't the tag under Skills also show up in search? This may be confusing to the average user.
